apps/api/prisma/migrations/20220712130328_create_grid_reference_and_service_customer/migration.sql

```diff
@@ -0,0 +1,56 @@
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- AlterTable
+ALTER TABLE [dbo].[ApplicationDetails] ADD [locationDescription] NVARCHAR(1000),
+[mapZoomLevel] NVARCHAR(1000),
+[firstNotifiedAt] DATETIME2,
+[submissionAt] DATETIME2;
+
+-- CreateTable
+CREATE TABLE [dbo].[ServiceCustomer] (
```

Comment:
so the technical name of the applicant is ServiceCustomer

Reply:
Exactly. This will include applicants and anyone else for other cases (not just applications)